### PR TITLE
Adding new monitor (Safe Liveness Expiration)

### PR DESCRIPTION
**Pull request by Ethnical**
**Description**
Liveness expiration module is a monitoring dedicated for the safes.
Ensuring that owners that operates the safes are still actives enough.
To do so we are monitoring multiples metrics like: 
- `blockTimestamp`: The block Timestamp of the latest block number on L1.
- `highestBlockNumber`: The lastest block number height on L1.
- `lastLiveOfAOwner`: Get the last activities for a given safe owner on L1.  
- `intervalLiveness`: the *interval* (in seconds) from the `LivenessModule` on L1.  

ab27497cea05fbd51b7b1c2ecde5bc69307ac0f27349f6bba4f3f21423116071

The alerts logic will be written into prometheus, thanks to the metrics we have right now.

### CODE REVIEW

**Comment by Ethnical** [comment]
@ajsutton  I just added a readme.md into the `drippie/` folder, with the previous informations to start standardized the main README. As go-reviewers is part of the `CODEOWNERS` on `Drippie/`. I need your small review there.

Also notice that the `secrets/` folder is not into the `CODEOWNERS` is that intended? 
```
*                       @ethereum-optimism/security-reviewers
op-monitorism/drippie   @ethereum-optimism/go-reviewers
```

File: README.md [hunk 1]
```diff
@@ -1,117 +1,100 @@
...
 
-## Monitors
+ ![ab27497cea05fbd51b7b1c2ecde5bc69307ac0f27349f6bba4f3f21423116071](https://github.com/ethereum-optimism/monitorism/assets/23560242/af7a7e29-fff5-4df3-82f0-94c2f28fde84)
 
-In addition the common configuration, each monitor also has their specific configuration
 
-* **Note**: The environment variable prefix for monitor-specific configuration is different than the global monitor config described above.
+The Liveness Expiration Monitor is made for monitoring the liveness expiration on Safes.
+
+| `op-monitorism/liveness_expiration` | [README](https://github.com/ethereum-optimism/monitorism/blob/aba37ff58b3503018ae5dbad5e5473af670834bc/op-monitorism/liveness_expiration/README.md) |
```
Comment (raffaele-oplabs) [inline README.md:24]: should be in the main branch, this is on a specific commit

File: README.md [hunk 2]
```diff
@@ -1,117 +1,100 @@
...
-   --l1.node.url value             [$FAULT_MON_L1_NODE_URL]         Node URL of L1 peer (default: "127.0.0.1:8545")                              
-   --l2.node.url value             [$FAULT_MON_L2_NODE_URL]         Node URL of L2 peer (default: "127.0.0.1:9545")                              
-   --start.output.index value      [$FAULT_MON_START_OUTPUT_INDEX]  Output index to start from. -1 to find first unfinalized index (default: -1) 
-   --optimismportal.address value  [$FAULT_MON_OPTIMISM_PORTAL]     Address of the OptimismPortal contract                                       
-```
+ The fault monitor checks for changes in output roots posted to the `L2OutputOracle` contract.
+ On change, reconstructing the output root from a trusted L2 source and looking for a match.
 
-On mismatch the `isCurrentlyMismatched` metrics is set to `1`.
+| `op-monitorism/fault` | [README](https://github.com/ethereum-optimism/monitorism/blob/aba37ff58b3503018ae5dbad5e5473af670834bc/op-monitorism/fault/README.md) |
```
Comment (raffaele-oplabs) [inline README.md:33]: should be in the main branch, this is on a specific commit

File: README.md [hunk 3]
```diff
@@ -1,117 +1,100 @@
...
-   --event.block.range value       [$WITHDRAWAL_MON_EVENT_BLOCK_RANGE]   Max block range when scanning for events (default: 1000) 
-   --start.block.height value      [$WITHDRAWAL_MON_START_BLOCK_HEIGHT]  Starting height to scan for events                       
-   --optimismportal.address value  [$WITHDRAWAL_MON_OPTIMISM_PORTAL]     Address of the OptimismPortal contract                   
-```
+ The withdrawals monitor checks for new withdrawals that have been proven to the `OptimismPortal` contract.
+ Each withdrawal is checked against the `L2ToL1MessagePasser` contract.
+
 
-If a proven withdrawal is missing from L2, the `isDetectingForgeries` metrics is set to `1`.
+| `op-monitorism/withdrawals` | [README](https://github.com/ethereum-optimism/monitorism/blob/aba37ff58b3503018ae5dbad5e5473af670834bc/op-monitorism/withdrawals/README.md) |
```
Comment (raffaele-oplabs) [inline README.md:42]: should be in the main branch, this is on a specific commit

File: README.md [hunk 4]
```diff
@@ -1,117 +1,100 @@
...
 ### Balances Monitor
 
 The balances monitor simply emits a metric reporting the balances for the configured accounts.
-```
-OPTIONS:
-   --node.url value                                             [$BALANCE_MON_NODE_URL]  Node URL of a peer (default: "127.0.0.1:8545") 
-   --accounts address:nickname [ --accounts address:nickname ]  [$BALANCE_MON_ACCOUNTS]  One or accounts formatted via address:nickname 
-```
+
+| `op-monitorism/balances` | [README](https://github.com/ethereum-optimism/monitorism/blob/aba37ff58b3503018ae5dbad5e5473af670834bc/op-monitorism/balances/README.md) |
```
Comment (raffaele-oplabs) [inline README.md:49]: should be in the main branch, this is on a specific commit

**Review by raffaele-oplabs — APPROVED** [review]

**Review by geoknee — APPROVED** [review]

**Pull request merged**